u-root.go

```diff
@@ -210,6 +211,7 @@ func parseFlags() {
 	base = flag.String("base", "", "Base archive to add files to")
 	useExistingInit = flag.Bool("useinit", false, "Use existing init from base archive (only if --base was specified).")
 	outputPath = flag.String("o", "", "Path to output initramfs file.")
+	initCmd = flag.String("initcmd", "", "Where to link /init to (default: bbin/init).")
```

Comment:
Why not just make the default /init? Because in the code above you're doing a check for it being not empty, but you can make sure it's the default value right here, which is clearer anyway.

Reply:
Are you sure? The default value calculated in pkg/uroot/uroot.go (TargetDir which is used in binary.go too) is based on BinaryDir.  Assigning the default value in u-root.go requires IMHO refactoring of the current path (in three files) in another commit and I am not sure what is the best way to do it.